railties/lib/rails/generators/test_unit/authentication/authentication_generator.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "rails/generators/test_unit"
+
+module TestUnit # :nodoc:
+  module Generators # :nodoc:
+    class AuthenticationGenerator < Rails::Generators::Base # :nodoc:
+      class_option :fixture, type: :boolean
+
+      hook_for :fixture_replacement
+
+      def create_user_test_file
+        template "test/models/user_test.rb"
+      end
+
+      def create_user_fixture_file
+        if options[:fixture] && options[:fixture_replacement].nil?
+          template "test/fixtures/users.yml"
+        end
```

Comment:
Let's drop all these options. We don't need you to be able to turn this off. You can just delete if you don't need it. Same with the fixture replacement.